### PR TITLE
Adds EmberJS Chennai Feb meetup videos

### PR DESCRIPTION
**Pull request by abhilashlr**
## What it does
This PR adds Chennai EmberJS Feb 29th 2020 meetup videos.

## Sources
https://www.youtube.com/playlist?list=PLh_rF0Qob_sxjhg9qPTlXn-6EZzmmTHoc

### CODE REVIEW

File: source/2020-03-13-the-ember-times-issue-139.md [hunk 1]
```diff
@@ -14,13 +14,11 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Feb 29th, 2020 - Chennai EmberJS Meetup 🇮🇳](https://www.youtube.com/playlist?list=PLh_rF0Qob_sxjhg9qPTlXn-6EZzmmTHoc)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+On February 29, Chennai EmberJS featured 7 speakers at their third Meetup: [@askjkrishna](https://twitter.com/askjkrishna), [@michaeldinesh9](https://twitter.com/michaeldinesh9), [@srajas02](https://twitter.com/srajas02), [@ahamedalthaf](https://github.com/ahamedalthaf), [@Kishore01kumar](https://twitter.com/Kishore01kumar), [@prasanth_lpk](https://twitter.com/prasanth_lpk), and [@ansankarganesh](https://twitter.com/ansankarganesh)! They covered various topics, including Web development - the past and the present, picking a framework: React vs Angular vs Vue vs Ember, Ember runloop, Ember model & relationships, frame communication with Ember, design systems using Storybook, and Finite state machines and Ember. 💖
```
Comment (ijlee2) [inline source/2020-03-13-the-ember-times-issue-139.md]: I think finite state machines can be all lowercase.
```suggestion
On February 29, Chennai EmberJS featured 7 speakers at their third Meetup: [@askjkrishna](https://twitter.com/askjkrishna), [@michaeldinesh9](https://twitter.com/michaeldinesh9), [@srajas02](https://twitter.com/srajas02), [@ahamedalthaf](https://github.com/ahamedalthaf), [@Kishore01kumar](https://twitter.com/Kishore01kumar), [@prasanth_lpk](https://twitter.com/prasanth_lpk), and [@ansankarganesh](https://twitter.com/ansankarganesh)! They covered various topics, including Web development - the past and the present, picking a framework: React vs Angular vs Vue vs Ember, Ember runloop, Ember model & relationships, frame communication with Ember, design systems using Storybook, and finite state machines and Ember. 💖
```

**Review by ijlee2 — APPROVED** [review]
Looks great, thanks for coordinating with the Chennai Meetup!

**Comment by abhilashlr** [comment]
@ijlee2 do you also think links to individual videos in a list with author makes it easy to read? I reused the Aug 3rd Meetup format. I’ll change if we all think list is a better way than continuous text.

**Comment by ijlee2** [comment]
Yeah, I think a list of talks will be easier to read and help a reader launch the video that they are interested in watching. Wanted to suggest that but wasn't sure what you might want the preceding text to say.

```markdown
Paragraph #1, with some text about to introduce the list:

- [Talk #1 title](url to video)
- [Talk #2 title](url to video)
etc.

Paragraph #2
```

(I'm on mobile, sorry if the code above doesn't make much sense.)

**Comment by abhilashlr** [comment]
> Yeah, I think a list of talks will be easier to read and help a reader launch the video that they are interested in watching. Wanted to suggest that but wasn't sure what you might want the preceding text to say.
> 
> ```gfm
> Paragraph #1, with some text about to introduce the list:
> 
> - [Talk #1 title](url to video)
> - [Talk #2 title](url to video)
> etc.
> 
> Paragraph #2
> ```
> 
> (I'm on mobile, sorry if the code above doesn't make much sense.)

Here's a change:
https://github.com/abhilashlr/ember-blog/pull/1/files?short_path=3a1c5d2#diff-3a1c5d21e1d21d9963534cac84c240d5

Please let me know if that is fine, I will add it to the base PR.

**Comment by abhilashlr** [comment]
@ijlee2 Done with the list. Thanks a lot for your feedback :)

File: source/2020-03-13-the-ember-times-issue-139.md [hunk 2]
```diff
@@ -14,13 +14,18 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Feb 29th, 2020 - EmberJS Chennai Meetup 🇮🇳](https://www.youtube.com/playlist?list=PLh_rF0Qob_sxjhg9qPTlXn-6EZzmmTHoc)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+On February 29th 2020, EmberJS Chennai featured seven speakers at their third Meetup. They covered various topics that included:
```
Comment (ijlee2) [inline source/2020-03-13-the-ember-times-issue-139.md:19]: I think the Markdown lint that we have requires a space in-between. This should help with passing CI.
```suggestion
On February 29th 2020, EmberJS Chennai featured seven speakers at their third Meetup. They covered various topics that included:

```

**Comment by abhilashlr** [comment]
Thanks a lot @ijlee2 🙏

**Pull request merged**